src/config.json

```
@@ -42,6 +42,7 @@
     "tokens": []
   },
   "alchemy": { "key": "cQFlLK8EokIGlJhd_soImwEyUoC7Ec8r" },
+  "walletConnect" : { "radicleBridge" : "https://radicle.bridge.walletconnect.org" },
```

```suggestion
  "walletConnect" : { "bridge" : "https://radicle.bridge.walletconnect.org" },
```
Radicle is implied here, and we only have one bridge anyway.